docs/content/docs/dev/table/catalogs.md

```diff
@@ -849,40 +850,34 @@ final TableEnvironment tableEnv = TableEnvironment.create(settings);
 In SQL Gateway, it is recommended to configure the settings in a yaml file so that all sessions can automatically
 use the pre-created Catalog. Usually, you need to configure the kind of Catalog Store and other
 required parameters for the Catalog Store.
+
 ```yaml
 table.catalog-store.kind: file
-table.catalog-store.file.path: /path/to/catalog/store/
+table.catalog-store.file.path: file:///path/to/catalog/store/
 ```
 
 ### Catalog Store Type
-Flink has two built-in Catalog Stores, namely GenericInMemoryCatalogStore and FileCatalogStore.
-Users can also customize their own Catalog Store.
+
+Flink has two built-in Catalog Stores, namely `GenericInMemoryCatalogStore` and `FileCatalogStore`.
+Although the catalog store model is extendable, so users can also implement their own custom Catalog Store.
 
 #### GenericInMemoryCatalogStore
-GenericInMemoryCatalogStore is an implementation of CatalogStore that saves configuration information in memory.
-All catalog configurations are only available within the session's lifecycle, and the stored catalog configurations will be
-automatically cleared after session reconstruction.
 
-<table class="table table-bordered">
-    <thead>
-      <tr>
-        <th class="text-left" style="width: 25%">Option</th>
-        <th class="text-center" style="width: 45%">Description</th>
-      </tr>
-    </thead>
-    <tbody>
-    <tr>
-      <td><h5>kind</h5></td>
-      <td>Specify the Catalog Store type to be used, which should be 'generic_in_memory'</td>
-    </tr>
-    </tbody>
-</table>
+`GenericInMemoryCatalogStore` is an implementation of `CatalogStore` that saves configuration information in memory.
+All catalog configurations are only available within the sessions' lifecycle, and the stored catalog configurations will be
+automatically cleared after the session is closed.
+
+{{< hint info >}} By default, if no catalog store related configuration is specified, the system uses this implementation.
+{{< /hint >}}
 
 #### FileCatalogStore
-FileCatalogStore can save the user's Catalog configuration to a file. To use FileCatalogStore, you need to specify the directory where the Catalog configuration
-needs to be saved. Different Catalogs will correspond to different files and each file will correspond to a Catalog Name.
 
-Here's an example directory structure representing the storage of Catalog configurations using FileCatalogStore:
+`FileCatalogStore` can save the Catalog configuration to a file. To use `FileCatalogStore`, you need to specify the directory where the Catalog configurations
+needs to be saved. Each Catalog will have its own file named the same as the Catalog Name.
+
+The `FileCatalogStore` implementation supports both local and remote file systems that are available via the [Flink `FileSystem` abstraction]({{< ref "docs/deployment/filesystems/overview" >}})
+
+Here is an example directory structure representing the storage of Catalog configurations using `FileCatalogStore`:
```

Comment:
We should explain the default behavior of FileCatalogStore here ?  

If the root directory does not exist, it will be created by default.   

Also, exception will be thrown and will It cause sql-gateway failed to start if the path does not exist and fails to be created.

Reply:
> We should explain the default behavior of FileCatalogStore here ?

If it is not explained how anybody will now how the feature works and what is possible? I think the main purpose of documentation is to offer a well organized and detailed infromation about the sw or feature, so the user is not destined to learn how things work by trial and error or checking the source code (which will take more time).

> If the root directory does not exist, it will be created by default.

That is not true right now, only if https://github.com/apache/flink/pull/23293 will be merged.

Reply:
I'm sorry, I may not have described it clearly. What I mean is that we should provide a more detailed description of the default behavior: including how FileCatalogStore handles the situation when a file does not exist.